<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Terminal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>796</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew_Tab"/>
    <addaction name="actionNew_Window"/>
    <addaction name="separator"/>
    <addaction name="action_Open_File_Manager"/>
    <addaction name="action_Close_Tab"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <action name="actionNew_Tab">
   <property name="text">
    <string>New &amp;Tab</string>
   </property>
   <property name="shortcut">
    <string>Meta+T</string>
   </property>
  </action>
  <action name="actionNew_Window">
   <property name="text">
    <string>New &amp;Window</string>
   </property>
   <property name="shortcut">
    <string>Meta+N</string>
   </property>
  </action>
  <action name="action_Open_File_Manager">
   <property name="text">
    <string>&amp;Open File Manager</string>
   </property>
  </action>
  <action name="action_Close_Tab">
   <property name="text">
    <string>&amp;Close Tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
